Makefile

```diff
@@ -11,12 +11,16 @@ SESSION_NAMESPACE = t4c-sessions
 EASE_DEBUG_PORT = 3390
 PORT ?= 8080
 
+CAPELLA_DOCKERIMAGES = $(MAKE) -C capella-dockerimages PUSH_IMAGES=1 LOCAL_REGISTRY_NAME=$(LOCAL_REGISTRY_NAME) LOCAL_REGISTRY_PORT=$(REGISTRY_PORT)
+
 # Adds support for msys
 export MSYS_NO_PATHCONV := 1
 
-build: backend frontend docs capella/remote readonly
+build: backend frontend docs
+	$(CAPELLA_DOCKERIMAGES) capella/remote capella/readonly
```

Comment:
I would spell it out, and not use an environment variable for an executable with arguments. (for readability mostly)

Reply:
Two reasons against it: 
1. The focus should be on the images that are built during in the step. With the current version, you can see them directly. With the new version, you have to read the whole (long) command first.
2. It's a long command and used in two different places. Would be code duplication. 

What do you think about these two points?

Reply:
Maybe we can find another name for it like `$(CAPELLA_DOCKERIMAGES_MAKE_COMMAND)`.

Reply:
I see, that's defined as part of the `CAPELLA_DOCKERIMAGES` env var. :+1: